docs/development/commits-branches-merging.md

```diff
@@ -37,13 +37,13 @@ base.
 
 In addition to `main`, the repository has three other long-lived branches:
 
-- `test` and `prod` correspond to the Test and Production [deploy environments](../deployment/README.md), respectively.
+- `test` and `prod` are read-only (no longer actively used) and correspond to the Test and Production [deploy environments](../deployment/README.md), respectively.
```

Comment:
Thanks for this note. Would you mind adding something like "The last release using these branches was `YYYY.MM.R`..." and linking to that release, just for completeness? (and so we have a quick reference later if we need).

Reply:
No problem! I added a short note and the link to the release in 66399e49c72e67b97f6b8a0cad5f1d9445fd6578